sfge/src/main/java/com/salesforce/graph/visitor/ApexPathWalker.java

```diff
@@ -274,6 +274,7 @@ private static final class ThrowStatementVertexVisitedException extends SfgeRunt
         private final ThrowStatementVertex vertex;
 
         private ThrowStatementVertexVisitedException(ThrowStatementVertex vertex) {
+            super(vertex.toString());
```

Comment:
This probably doesn't need telemetry. We use this exception to help with walking paths and note down when a path ends in a throws statement. More of a functional thing than an error state.

Reply:
Understood.